_docs/flavour/expression-language.md

```diff
@@ -34,25 +34,25 @@ EL consists of the following elements:
 
 # `this` object
 
-As you [already know](templates.html#creating-a-new-page), page consists of HTML template and view class.
-To refer to instance of view class from an HTML template, you can use `this` literal.
-Like in Java, you are not forced to do it.
+As you [already know](templates.html#creating-a-new-page), a page consists of an HTML template and a view class.
+To refer to an instance of the view class from an HTML template, you can use `this` literal.
+Like in Java, you are not required to do it.
```

Comment:
Why don't you like 'force' word?

Reply:
LOL!  "Forced" sounds like somebody is standing behind you, ordering you to type "this."  :-)

Required is less personal -- the language syntax enforces it.

Not a big deal -- Let me know how you want it and I'll change the PR how you want.

Reply:
No, it's ok. I just wanted to know what's the reason